Shop-1/src/routes/productsRouter.ts

```diff
@@ -0,0 +1,20 @@
+import { express } from "express";
+import {
+  postProduct,
+  getProducts,
+  getProduct,
+  updateProduct,
+  deleteProduct,
+  getProductParams,
+} from "../controllers/products";
+import routes from "./usersRouter";
+
+const routes = routes();
```

Comment:
Look at how you did this in the userRoutes file. You need to call express when you make a router.

const routes = express.Router();

I like to name them userRoutes and productRoutes instead of naming them all routes to help keep them straight.

Reply:
oh yeah i changed that after i submitted it